python/lsst/ts/Dome/mock_controller.py

```diff
@@ -246,6 +246,9 @@ async def request_and_send_status(self, llc, llc_name):
         llc_name: LlcName
             The name of the Lower Level Component.
         """
+        self.log.debug("Determining current TAI.")
+        await self.determine_current_tai()
```

Comment:
Where this method is defined (a few lines past this block of changess): please modify the doc string:
* Say that it returns the current TAI (right now it talks about a time difference, which is no longer true)
* The method exists so you can replace it with a mock for unit tests.

(This assumes you are still finding it useful to mock the method. If not, please consider getting rid of it.)